modules/data.atmosphere/R/download.MACA.R

```diff
@@ -0,0 +1,145 @@
+##' Download MACA CMIP5 outputs for a single grid point using OPeNDAP and convert to CF 
+##' @name download.MACA
+##' @title download.MACA
+##' @export
+##' @param outfolder
+##' @param start_date
+##' @param end_date
+##' @param lat
+##' @param lon
+##' @param model , select which MACA model to run (options are CM3, ESM2M, ESM2G)
```

Comment:
I'd say that selecting multiple models needs to be handled higher up in the workflow, and the workflow should loop over models at a high level.
For met ensembles this needs more discussion, since we never really finalized how we're storing driver and IC uncertainty more generally. That decision has a lot to do with whether ensembles are handled within the functions or whether we're looping over ensemble members.